server/model/account.js

```diff
@@ -16,7 +16,20 @@ const AccountSchema = new Schema({
 		gender: String,
 		phoneNumber: String,
 	},
-	profilePhoto: PhotoSchema,
+	profilePhoto: {
+		photoData: {
+			type: Buffer, // Storing the image data directly in MongoDB as a binary buffer
+			required: false,
+		},
+		photoContentType: {
+			type: String, // Storing the file's MIME type
+			required: false,
+		},
+		uploadedAt: {
+			type: Date,
+			default: Date.now,
+		},
+	},
```

Comment:
this file does not need to be changed, probably some issues when merging from main

Reply:
Oh my bad, I reverted the changes, should be fine now.